crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -145,19 +154,31 @@ impl ActionHandler for BridgeLockAction {
     }
 }
 
-/// returns the length of a serialized `Deposit` message.
-pub(crate) fn get_deposit_byte_len(deposit: &Deposit) -> u128 {
-    use prost::Message as _;
-    let raw = deposit.clone().into_raw();
-    raw.encoded_len() as u128
+/// Returns a modified byte length of the deposit event. Length is calculated with reasonable values
+/// for all fields except `asset` and `destination_chain_address`, ergo it may not be representative
+/// of on-wire length.
+pub(crate) fn calculate_base_deposit_fee(deposit: &Deposit) -> Option<u128> {
+    let variable_length = deposit
+        .asset
+        .to_string()
+        .len()
+        .checked_add(deposit.destination_chain_address.len())
+        .expect("deposit byte length overflowed usize") as u128;
```

Comment:
Options allow for the `?` operator. Please use that here.

This whole expression could then become something like:

```rust
let variable_len = deposit.asset.display_len().checked_add(deposit.destination_chain_address.len())?;
DEPOSIT_BASE_FEE.checked_add(variable_len)
```

Alternatively everything chained:

```rust
deposit.asset
    .display_len()
    .checked_add(deposit.destination_chain_address.len())
    .and_then(|var_len| DEPOSIT_BASE_FEE.checked_add(var_len))
```